### PR TITLE
structur and timer function

### PR DESCRIPTION
**Pull request by iwanast**
Structur and timer function

### CODE REVIEW

File: scripts/game.js [hunk 1]
```diff
@@ -1,19 +1,92 @@
...
+    // Will need to save the time when the game is started into a variable
     //Then save the time when the game ends into a second variable
     //Then compare the difference between the variables
     //Then adjust for penalties
-    //Then save the final time into the local storage
+    //Then save the final time into the local storage by a new function (Ben working on)
 }
 
+function penaltyNumber(){
+    if (difficulty == "easy"){
```
Comment (BenCiantar) [inline scripts/game.js:50]: Should we swap this so there's less of a penalty for missing a rabbit or misclicking on easy than hard?

File: scripts/game.js [hunk 2]
```diff
@@ -1,19 +1,92 @@
...
-    //Hard one. Will need to save the time when the game is started into a variable
+    finalTime = 0;
+    for(let i = 0; i < rabbitAppearTime.length; i++){
+        if(rabbitClickedTimes[i] != null){
+            finalTime += (rabbitClickedTimes[i].getTime() - rabbitAppearTime[i].getTime());
+        }
+    }
+    //Depending on how much we want the penaltys to be depending on difficulty?*************************
+    finalTime += (rabbitPenalty * penaltyNumber()); // example with "easy" 2000 milliseconds 
+    finalTime += (misclickPenalty * penaltyNumber()); // should we set this different??
```
Comment (BenCiantar) [inline scripts/game.js:32]: I think we discussed having a lower penalty for a misclick than missing a rabbit. I think that would make hard mode more balanced.

File: scripts/structure.js [hunk 3]
```diff
@@ -143,17 +140,17 @@ function displaySummary() {
     </section>`;
 }
 
-function calculateAverageReaction(array) {
-  let avReactTime = 0;
+// function calculateAverageReaction(array) {
```
Comment (BenCiantar) [inline scripts/structure.js:143]: Commented out for testing?

File: scripts/game.js [hunk 4]
```diff
@@ -1,19 +1,92 @@
 //Game functions
-let rabbitAppearTime = []; //Empty array that we can fill with the times each rabbit appeared
-let rabbitClickedTimes = []; //Empty array we can fill with the time each rabbit was clicked
+let time1 = new Date(2021, 9, 13, 22, 10, 30, 100); console.log(time1);
```
[the remark this answers is not in the document]
Comment (iwanast) [reply scripts/game.js:2]: Yes

**Review by BenCiantar — APPROVED** [review]

**Pull request merged**